src/heap.h

```diff
@@ -281,7 +281,9 @@ class ObjectHeap final : public Heap {
   ByteArray* allocate_proxy(int length, uint8* memory, bool dispose = false) {
     return allocate_external_byte_array(length, memory, dispose, false);
   }
-  ByteArray* allocate_proxy() { return allocate_proxy(0, null); }
```

Comment:
This would be so much more readable if it took an enum instead of a bool, so you could write Heap::DISPOSE as the argument.

Reply:
Yeah, that would be nicer. We should fix that.